src/components/DateTimePicker/Internal/OcPicker.tsx

```diff
@@ -83,18 +86,23 @@ function InnerPicker<DateType>(props: OcPickerProps<DateType>) {
     popupPlacement,
     popupStyle,
     shape = DatePickerShape.Rectangle,
+    showNow,
+    showOk,
     showTime,
+    showToday,
```

Comment:
is this effectively "highlightToday"?

Reply:
No, it should be `showTodayButton`. But needed to align the naming for now. I'll note it in the docs.